simplq/src/components/pages/Admin/admin.module.scss

```diff
@@ -8,6 +8,26 @@
   height: max-content;
 }
 
+.share {
+  .button-background {
+    background-color: $primary-color-dark;
+    border-radius: 0.6rem !important;
+  }
```

Comment:
Is this !important required? I didn't notice any problems without it.

Reply:
I will update you now. Let complete and close this PR today

Reply:
Yes !important is not required now. When I started this issue I noticed some other border-radius CSS already declared so included !important to overide it. After I have changed some config !important is not required but I forgot to remove it.